app/models/manageiq/providers/workflows/automation_manager/workflow_instance.rb

```diff
@@ -66,6 +66,7 @@ def run(zone: nil, role: "automation", object_type: nil, object_id: nil)
       end
     end
 
+    Floe::Workflow::Runner.docker_runner = floe_docker_runner
```

Comment:
NOTE I'm not sure the best place to set this, maybe an `Engine` initializer?

Reply:
yeah it feels like this can be a one-time setup thing - not sure where it should live - perhaps a class method like `init_docker_runner`, which only runs once?  We do set up the Floe logger in an Engine initializer, so perhaps that's a good spot.

Reply:
I forgot when I said this that plugins support _normal_ config/initializers, but as I said in the other comment, it would run on any initialization - not sure if we can even delay load it though 🤷

Reply:
Yeah we're calling all of the `Vmdb::Plugins.init_*` on every initialization anyway, I'll move it to `config/initializers` just wish there was a better place to put this that wasn't a.) on every init or b.) all the way down in WorkflowInstance#execute

Reply:
Agreed - we need some sort of way to delay load things but run a one-time initialization when that thing is loaded.  🤔